server/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "SUP server for web and API",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
```

Comment:
Tests do not work me, it's complaining about ES modules.

Reply:
should work now... had to add transformation for jest ts files